feature/src/main/java/org/woowatechcamp/banchan/feature/purchase/orderdetail/OrderDetailFragment.kt

```diff
@@ -29,6 +37,13 @@ class OrderDetailFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        initAdapter()
+    }
+
+    private fun initAdapter() {
+        with(binding) {
+            rvOrderDetail.adapter = adapter
+        }
```

Comment:
binding내부에 한 개만 감싸지니까,

```suggestion
        binding.rvOrderDetail.adapter = adapter
```
라고만 해도 괜찮을 것 같습니다!

Reply:
아직 adapter 생성만 해놓은 거라 나중에 데이터 관련 로직 추가되면 `with`로 감쌀 수도 있을 것 같아서 일단은 저렇게 작성해 놓았습니다. 
데이터 로직 추가했을 때 필요 없다면 수정해두겠습니다.